GitUI/Editor/FileViewerInternal.cs

```diff
@@ -165,7 +165,11 @@ public void SetText(string text, bool isDiff = false)
 
         public void SetHighlighting(string syntax)
         {
```

Comment:
arguments must be validated on all public API
if `syntax` is `null`, `""` or `"   "` there is no reason to proceed further
the only thing you may wish to do in this case (before exiting) is to reset any existing highlighting to defaults